exec.go

```diff
@@ -101,7 +101,8 @@ following will output a list of processes running in the container:
 		if err == nil {
 			os.Exit(status)
 		}
-		return fmt.Errorf("exec failed: %w", err)
+		fatalWithCode(fmt.Errorf("exec failed: %w", err), -1)
```

Comment:
> -1

Can't we use `255` here?

Reply:
Yes, it will probably look a tad better.

Fixed, PTAL @AkihiroSuda

Reply:
Doesn't seem fixed

Reply:
My bad. Let's try again